cxx/python/seismic/seismic_py.cc

```diff
@@ -593,6 +593,174 @@ PYBIND11_MODULE(seismic, m) {
     .def("__setitem__",py::overload_cast<const std::string,const std::string>(&BasicMetadata::put))
     .def("__setitem__",py::overload_cast<const std::string,const py::object>(&Metadata::put_object))
   ;
+  py::class_<LoggingEnsemble<Seismogram>, Ensemble<Seismogram> >(m,"SeismogramEnsemble","Gather of vector(3c) time series objects")
+    .def(py::init<>())
+    .def(py::init<const size_t >())
+    .def(py::init<const Metadata&, const size_t>())
+    .def(py::init<const Ensemble<Seismogram>&>())
+    .def(py::init<const LoggingEnsemble<Seismogram>&>())
+    .def("kill",&LoggingEnsemble<Seismogram>::kill,"Mark the entire ensemble dead")
+    .def("live",&LoggingEnsemble<Seismogram>::live,"Return true if the ensemble is marked live")
+    .def("dead",&LoggingEnsemble<Seismogram>::dead,"Return true if the entire ensemble is marked dead")
+    .def("validate",&LoggingEnsemble<Seismogram>::validate,"Test to see if the ensemble has any live members - return true of it does")
+    .def("set_live",&LoggingEnsemble<Seismogram>::set_live,"Mark ensemble live but use a validate test first")
+    .def_readwrite("elog",&LoggingEnsemble<Seismogram>::elog,"Error log attached to the ensemble - not the same as member error logs")
+    .def(py::pickle(
+      [](const LoggingEnsemble<Seismogram> &self) {
+        pybind11::gil_scoped_acquire acquire;
+        try{
+          pybind11::object sbuf;
+          sbuf=serialize_metadata_py(self);
+          stringstream sselog;
+          boost::archive::text_oarchive arelog(sselog);
+          arelog << self.elog;
+          pybind11::module pickle = pybind11::module::import("pickle");
+          pybind11::object dumps = pickle.attr("dumps");
+          size_t nmembers=self.member.size();
+          py::list dlist;
+          for(auto i=0;i<nmembers;++i)
```

Comment:
Loop here is not necessary. As mentioned in #207, `pickle.dumps(list(self.member))` here should be supported.